cmd/nginx-ingress/flags.go

```diff
@@ -331,11 +340,31 @@ func validationChecks() {
 	}
 }
 
+// validateNamespaceNames validates the namespaces are in the correct format
```

Comment:
Can we get a unit test for this if possible? for cases like:
`-watch-namespace=watched_namespace,`
`-watch-namespace="watched_namespace1,watched_namespace2"`
`-watch-namespace="watched_namespace1","watched_namespace2"`

It think there are regex checks in place for invalid chars already